.github/workflows/pdf-build.yaml

```diff
@@ -83,26 +83,36 @@ jobs:
           restore-keys: |
             pdf-${{ inputs.build_type }}-${{ inputs.language }}-
 
+      - name: Cache Inkscape artifacts
+        if: steps.cache-pdf.outputs.cache-hit != 'true'
+        uses: actions/cache@v4
+        with:
+          path: |
+            ./svg-inkscape
+          key: >-
+            inkscape-${{ hashFiles('assets/glyphs/*') }}
+          restore-keys: |
+            inkscape-
```

Comment:
This would work as long as svg files don't contain any language-specific text, right?

Reply:
Correct. AFAIK, they don't.